xbmc/pvr/PVRDatabase.cpp

```diff
@@ -45,7 +45,7 @@ void CPVRDatabase::CreateTables()
 {
   CLog::Log(LOGINFO, "PVR - %s - creating tables", __FUNCTION__);
 
-  CLog::Log(LOGDEBUG, "PVR - %s - creating table 'channels'", __FUNCTION__);
+  CLog::Log(LOGDEBUG, "CPVRDatabase - %s - creating table 'channels'", __FUNCTION__);
```

Comment:
How should I know with "PVR: creating table 'channels'" that is is located in CDatabase.cpp?

As noted above, I dont have your overview. You may know directly where it is located. I can only say from my point of view: It was so much helpful for me when I found the classname (wich is mostly the filename) in the debug log.

But: I don't hold on this point - if this is an issue for you its quickly removed - sad for me but I've no problem with it.